src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/ApplicationSummary.vue

```diff
@@ -45,5 +154,85 @@ export default defineComponent({
       ],
     };
   },
+  computed: {
+    step2ReferenceDocumentIcon() {
+      switch (this.step2Progress) {
+        case this.IN_PROGRESS:
+          return "mdi-arrow-right";
+        case this.COMPLETE:
+          return "mdi-check";
+        default:
+          return "";
+      }
+    },
+    step2ReferenceDocumentText() {
+      switch (this.step2Progress) {
+        case this.IN_PROGRESS:
+          return "In Progress";
+        case this.COMPLETE:
+          return "Complete";
+        default:
+          return "";
+      }
+    },
+    step2Progress() {
+      switch (this.applicationStatus?.status) {
+        case "Submitted":
+        case "InProgress":
+          return this.IN_PROGRESS;
+        case "Ready":
+        case "PendingQueue":
+        case "Escalated":
+          return this.COMPLETE;
+        default:
+          return "";
+      }
+    },
+    step3RegistryAssessmentIcon() {
+      switch (this.step3Progress) {
+        case this.NOT_STARTED:
+          return "";
+        case this.IN_PROGRESS:
+          return "mdi-arrow-right";
+        default:
+          return "";
+      }
+    },
+    step3RegistryAssessmentText() {
+      switch (this.step3Progress) {
+        case this.NOT_STARTED:
+          return "Not yet started";
+        case this.IN_PROGRESS:
+          return "In progress";
+        default:
+          return "";
+      }
+    },
+    step3Progress() {
+      switch (this.applicationStatus?.status) {
+        case "Submitted":
+        case "InProgress":
+          return this.NOT_STARTED;
+        case "Ready":
+        case "PendingQueue":
+          return this.IN_PROGRESS;
+        default:
+          return "";
+      }
+    },
+  },
+  methods: {
+    goTo(id: string | undefined) {
+      this.alertStore.setSuccessAlert("not implemented yet this will go to another route " + id);
+    },
+  },
 });
 </script>
+<style scoped>
+.border-top {
```

Comment:
vuetify class `border-t-md` should do it

Reply:
annoyingly so if I used border-t-md the top border would change to a light grey instead of black as requested by the design team. 

with border-t-md
![image](https://github.com/bcgov/ECC-ECER/assets/74216496/e756e803-ecab-4df4-91f3-7127f97a5b1e)

custom class
![image](https://github.com/bcgov/ECC-ECER/assets/74216496/a7acc0bd-bb92-4095-9136-ce0bcfc52ff4)